resource-managers/mesos/src/main/scala/org/apache/spark/deploy/mesos/ui/DriverPage.scala

```diff
@@ -0,0 +1,180 @@
+/*
```

Comment:
Likewise this isn't part of the backport is it? https://github.com/apache/spark/pull/17686/files

Reply:
I'll look into this as well...

Reply:
I'm not sure what I did to make this whole file look new, but I've copied the 1.6 current and reapplied stripXSS locally.  Waiting for my build to pass to commit again.